core/internals.h

```diff
@@ -74,8 +76,11 @@
 #define URI_REGISTRATION_SEGMENT_LEN    2
 #define URI_BOOTSTRAP_SEGMENT           "bs"
 #define URI_BOOTSTRAP_SEGMENT_LEN       2
+#define URI_DELETE_ALL_SEGMENT          "/"
+#define URI_DELETE_ALL_SEGMENT_LEN      1
```

Comment:
I think the bug is coming from leshan:
https://github.com/eclipse/leshan/blob/master/leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/BootstrapResource.java#L102

We also need to fix that for leshan bootstrap server